serverless/knative_serving/serverless-metering.adoc

```diff
@@ -8,11 +8,11 @@ toc::[]
 
 As a cluster administrator, you can use metering to analyze what is happening in your {ServerlessProductName} cluster.
 
-For more information about metering on {product-title}, see xref:../metering/metering-about-metering.adoc#about-metering[About metering].
+For more information about metering on {product-title}, see xref:../../metering/metering-about-metering.adoc#about-metering[About metering].
```

Comment:
To me, "For more information about metering on OpenShift" sounds odd. Maybe "...about {product-title} metering" or "about using metering on {product-title}..."?